apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -72,10 +71,10 @@ class DanceVisualizationColumn extends React.Component {
   };
 
   /*
-    Turn the song filter off unless there is a teacher override
+    Turn the song filter off
   */
   turnFilterOff() {
-    this.setState({filterOff: queryString.parse(window.location.search).songfilter !== 'on'});
+    this.setState({filterOff: true});
```

Comment:
queryString check is no longer necessary - we pass this function to AgeDialog and the first thing we do in AgeDialog is check this.